src/main/java/org/opentripplanner/routing/algorithm/raptor/transit/AccessEgress.java

```diff
@@ -29,6 +30,11 @@ public int stop() {
     return toFromStop;
   }
 
+  @Override
+  public int cost() {
+    return RaptorCostConverter.toRaptorCost(lastState.getWeight());
```

Comment:
I think we should cash these values in this class - I will guess this is typically called around 2 500 000 times during on search normal search. 100 access/egress  and 6-8 hours search-window and avg. 5 paths reaching each access/egress stop. I would have to run a test to get the actual number, but this is part of the absolute inermost loop of the computation - the fact that it is "only" access and egress helps, but not as much as one at first might think.

Reply:
I've updated the implementation to use a cached value.